src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 
-export const TodoList: React.FC = () => (
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  selectedTodoId: number | null;
+  onSelectedTodoId: (todo: number) => void;
+}
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  selectedTodoId,
+  onSelectedTodoId,
```

Comment:
```suggestion
  onSelect,
```

(or `onTodoSelect`)

your name sounds too complicated